### PR TITLE
Update spirv.h to the latest version in SPIRV-Headers.

### PR DESCRIPTION
**Pull request by dnovillo**
This adds support for the tile image extension.

### CODE REVIEW

**Comment by dnovillo** [comment]
@chaoticbob This is failing the clang-format check because of the added `spirv.h` file.  Should we disable clang-format checks on this file?  I can re-format it, but it would create a significant diff against upstream.

**Comment by dnovillo** [comment]
Fixed clang formatting issues.  @chaoticbob PTAL.  Thanks.

File: common/output_stream.cpp [hunk 1]
```diff
@@ -279,6 +279,8 @@ std::string ToStringSpvDim(SpvDim dim) {
       return "Buffer";
     case SpvDimSubpassData:
       return "SubpassData";
+    case SpvDimTileImageDataEXT:
+      return "SpvDimTileImageDataEXT";
```
Comment (chaoticbob) [inline common/output_stream.cpp:283]: I think generally the `Spv` prefix is dropped.

**Review by chaoticbob — CHANGES_REQUESTED** [review]

File: common/output_stream.cpp (hunk 1 above)
Comment (dnovillo) [reply common/output_stream.cpp]: > I think generally the `Spv` prefix is dropped.

Thanks.  Fixed.

**Review by chaoticbob — APPROVED** [review]
LGTM

**Pull request merged**